clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -16,29 +16,46 @@
 
 package com.netflix.spinnaker.clouddriver.azure.client
 
+import com.microsoft.azure.CloudException
+import com.microsoft.azure.credentials.ApplicationTokenCredentials
+import com.microsoft.azure.management.resources.ResourceManagementClientImpl
 import com.microsoft.azure.management.resources.models.Deployment
 import com.microsoft.azure.management.resources.models.DeploymentExtended
 import com.microsoft.azure.management.resources.models.DeploymentMode
 import com.microsoft.azure.management.resources.models.DeploymentOperation
-import com.microsoft.azure.management.resources.models.DeploymentOperationsListParameters
-import com.microsoft.azure.management.resources.models.DeploymentOperationsListResult
 import com.microsoft.azure.management.resources.models.DeploymentProperties
 import com.microsoft.azure.management.resources.models.ResourceGroup
 import com.microsoft.azure.management.resources.ResourceManagementClient
-import com.microsoft.azure.management.resources.ResourceManagementService
-import com.microsoft.windowsazure.exception.ServiceException
 import com.netflix.spinnaker.clouddriver.azure.common.AzureUtilities
 import com.netflix.spinnaker.clouddriver.azure.security.AzureCredentials
 import groovy.transform.Canonical
 import groovy.transform.CompileStatic
+import okhttp3.logging.HttpLoggingInterceptor
 
 @CompileStatic
 class AzureResourceManagerClient extends AzureBaseClient {
 
-  AzureResourceManagerClient(String subscriptionId) {
+  private final ResourceManagementClient client
+
+  /**
+   *
+   * @param subscriptionId
```

Comment:
Would be more consistent to either drop the javadoc comment altogether, or include both parameters.